Publicize: Improve primary form component. (#21090)
Automattic/jetpack: projects/plugins/jetpack/extensions/plugins/publicize/hooks/use-select-social-media/Readme.md

# useSelectSocialMedia hook

> React hook to get relevant data about the social media connections.
> You'd like to use together with the [useSocialMediaConnections](../use-social-media-connections) hook.

Reviewer: ```suggestion
React hook to get relevant data about the social media connections.
This hook should be used together with the [useSocialMediaConnections](../use-social-media-connections) hook.
```
These 2 sentences are a little confusing to me. Do you mean "This hook should be used together with"?
